dsgrid/api/app.py

```diff
@@ -135,7 +136,7 @@ async def list_project_dimensions(project_id: str):
     for item in project.get_dimension_query_names_model().dict().values():
         dimension = create_project_dimension_model(project.get_dimension(item["base"]).model, True)
         dimensions.append(dimension)
-        for query_name in item["supplemental"]:
+        for query_name in itertools.chain(item["subset"], item["supplemental"]):
```

Comment:
Having subset dimensions show up here is debatable. Their only potential use is to show what dimensions would be available for derived datasets.

Reply:
What are the cons?

Reply:
It might be presenting information and decisions to the user that are distracting and unhelpful.